tasks/configure.yml

```diff
@@ -4,39 +4,42 @@
     src: "alert.rules.j2"
     dest: "{{ prometheus_config_dir }}/rules/ansible_managed.rules"
     owner: root
-    group: prometheus
+    group: "{{ prometheus_id }}"
```

Comment:
What is `prometheus_id` and how it is obtained. If this is something created in runtime then it should be prefixed with `__`.

Reply:
it should be named prometheus_uid, and it's the uid of prometheus inside the container; should be the same inside and outside a container